MAINTAINERS.md

```diff
@@ -16,3 +16,4 @@ This document contains a list of maintainers in this repo. See [opensearch-proje
 | Vamshi Vijay Nakkirtha  | [vamshin](https://github.com/vamshin)                 | Amazon      |
 | Vijayan Balasubramanian | [VijayanB](https://github.com/VijayanB)               | Amazon      |
 | Yuncheng Lu             | [luyuncheng](https://github.com/luyuncheng)           | Bytedance   |
+| Tejas Shah              | [shatejas](https://github.com/shatejas)               | Amazon      |
```

Comment:
nit: I believe we are adding based on alphabetical order.

Reply:
really? I will fix it.